code/modules/jobs/job_types/chaplain.dm

```diff
@@ -9,6 +9,9 @@
 	supervisors = "the head of personnel"
 	selection_color = "#dddddd"
 	chat_color = "#8AB48C"
+	minimal_player_age = 1
+	exp_requirements = 120
+	exp_type = EXP_TYPE_CREW
```

Comment:
???????

Reply:
Because chaplain has duties on a cult round, and also because they have easy access to both a pretty good weapon and powerful armor.

High-shittery inclined role.

If you disagree I'm not particularly opposed to dropping this one back to open access however.

Reply:
Dropping chaplain back down just so I can mark this as resolved since there's been no further reply.